### PR TITLE
deliverable 1

### PR DESCRIPTION
**Pull request by Mike-Ebert**
First page of Monday issue

### CODE REVIEW

File: 1906-11-12.xml [hunk 1]
```diff
@@ -0,0 +1,1438 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/dig-eg-gaz/resources/master/out/egSchema.rnc" type="application/relax-ng-compact-syntax"?>
+<?xml-stylesheet type="text/xsl" href="dig-eg-gaz.xsl"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title level="m" type="main">Digital Egyptian Gazette</title>
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor>Mihael Ebert</editor>
```
Comment (whanley) [inline 1906-11-12.xml]: spelling of your name!

File: 1906-11-12.xml [hunk 2]
```diff
@@ -0,0 +1,1438 @@
...
+                        <p>Prices from 10/6</p>
+                        <p>Sold by all Stationers and Importers in Cairo and Alexandria.</p>
+                        <p>Catalogue free.</p>
+                        <p>Mable, Todd &amp; Bard, High Holborn, London.</p>
+                    </div>
+                </div>
+                <div type="section" scope="advertisement"> 
+                    <cols n="3"/>
+                    <cb n="1"/>
+                </div>  <div type="advert" xml:id="deg-ad-orm01">
```
Comment (whanley) [inline 1906-11-12.xml]: This `</div>` tag, which closes the advertising section, should come after all of these ads, just before the end of the page.

Comment (Mike-Ebert) [reply 1906-11-12.xml]: Where exactly does this tag go? I tried putting it right after the last ad and it turned the box red. I also tried putting it at the very bottom of the page and it also turned the box red.

Comment (whanley) [reply 1906-11-12.xml]: As long as your remove the tag from line 61, it should work find in both places you list. I can take a look in class tomorrow.

File: 1906-11-12.xml [hunk 3]
```diff
@@ -0,0 +1,1439 @@
...
+                        </row>
+                        <row>
+                            <cell>Interchangeable return tickets with the Austrian Lloyd Cy. (available
+                                one way by Messageries</cell>
+                            <cell>£21.11.10</cell>
+                            <cell>£15.11.2</cell>
+                        </row>
+                    </table>
+                    <table rend="frame" xml:id="SailingfromPortSaid">
+                        <head><hi rend="bold">Sailing from Port Said in July, 1905</hi></head>
```
Comment (whanley) [inline 1906-11-12.xml]: 1905?

File: 1906-11-12.xml [hunk 4]
```diff
@@ -0,0 +1,1439 @@
...
+                            <cell>Probably on</cell>
+                            <cell>Sunday</cell>
+                            <cell><date when="1906-11-18">18 November</date></cell>
+                            <cell>Ville de la Ciatat</cell>
+                            <cell>Capt. Etienne</cell>
+                            <cell>returning from Australia</cell>
+                        </row>
+                    </table>
+                    <table rend="frame" xml:id="SailingfromSuez">
+                        <head><hi rend="bold">Sailing from Suez in July, 1905</hi></head>
```
Comment (whanley) [inline 1906-11-12.xml]: July 1905?

File: 1906-11-12.xml [hunk 5]
```diff
@@ -0,0 +1,1439 @@
...
+                            <cell>BRITISH PRINCE</cell>
+                            <cell>due from</cell>
+                            <cell><placeName>Antwerp</placeName> &amp; <placeName>London</placeName></cell>
+                            <cell><date when="1906-11-05">December 5</date></cell>
+                        </row>
+                        <row>
+                            <cell>INDIAN PRINCE</cell>
+                            <cell>due from</cell>
+                            <cell><placeName>Manchester</placeName></cell>
+                            <cell><date when="1905-07-29">July 29</date></cell>
```
Comment (whanley) [inline 1906-11-12.xml]: date?

File: 1906-11-12.xml [hunk 6]
```diff
@@ -0,0 +1,1439 @@
...
+                            <cell><name>Princess Alice</name></cell>
+                            <cell><measure quantity="5026" unit="ton">5026</measure> Tons</cell>
+                            <cell>about <date when="1906-11-30">30 November</date></cell>
+                        </row>
+                        <row>
+                            <cell><name>Roon</name></cell>
+                            <cell><measure quantity="8881" unit="ton">8881</measure> Tons</cell>
+                            <cell>about <date when="1906-12-14">14 December</date></cell>
+                        </row>
+
```
Comment (whanley) [inline 1906-11-12.xml]: 1906

File: 1906-11-12.xml [hunk 7]
```diff
@@ -0,0 +1,1439 @@
...
+                            <cell>No. 135</cell>
+                            <cell>(Building)</cell>
+                        </row>
+                    </table>
+                    <p>*Second class accommodation only, unless specially reserved.—Fares : Alexandria
+                        to Liverpool, 1st, £14 Single, £25 Return. 2nd, £9 Single, £15 Return.—To Malta,
+                        1st, £5 Single, £9 Return, 2nd, £3 Single, £5 Return.—Return tickets available
+                        for six months.</p>
+                    <p><name>S.S. Seti</name> now on the berth, will sail on or about <date
+                        when="1905-07-17">Monday, July 17</date>, to be followed by <name>S.S.
```
Comment (whanley) [inline 1906-11-12.xml]: date?

File: 1906-11-12.xml [hunk 8]
```diff
@@ -0,0 +1,1853 @@
...
+                        <row>
+                            <cell><name>Princess Alice</name></cell>
+                            <cell><measure quantity="5026" unit="ton">5026</measure> Tons</cell>
+                            <cell>about <date when="1906-11-30">30 November</date></cell>
+                        </row>
+                        <row>
+                            <cell><name>Roon</name></cell>
+                            <cell><measure quantity="8881" unit="ton">8881</measure> Tons</cell>
+                            <cell>about <date when="1906-12-14">14 December 1906</date></cell>
+
```
Comment (whanley) [inline 1906-11-12.xml]: this `</div>` tag closes page 2 as soon as you start it--move the closing tag to the end of the page (i.e. just before `<div type="page" n="3">`

**Comment by whanley** [comment]
Deliverable 1 complete.

**Pull request merged**